### PR TITLE
Dump loaded `lang_item`s from hir

### PR DESCRIPTION
**Pull request by catap**
Here a two places where it should be dump:
 - inside `dump_hirfile`;
 - inside `from_ast.cpp`.

### CODE REVIEW

**Comment by thepowersgang** [comment]
Good idea, although the wording of the messages needs a little tweaking - and the `dump_hirfile` output should include the path as well as the name.

**Comment by catap** [comment]
@thepowersgang any suggestion for name tweaking ?

File: src/hir/from_ast.cpp [hunk 1]
```diff
@@ -2242,6 +2243,7 @@ ::HIR::CratePtr LowerHIR_FromAST(::AST::Crate crate)
             const auto& name = lang.first;
             const auto& path = lang.second;
             auto irv = rv.m_lang_items.insert( ::std::make_pair(name, path) );
+            DEBUG("Load language item '" << lang.first << "' at " << lang.second << " for " << ext_crate.first);
```
Comment (thepowersgang) [inline src/hir/from_ast.cpp:2246]: "from" not "for" would be more suitable.

File: tools/dump_hirfile/main.cpp [hunk 2]
```diff
@@ -86,6 +86,11 @@ void Dumper::dump_crate(const char* name, const ::HIR::Crate& crate) const
     }
 #endif
 
+    for(const auto& i : crate.m_lang_items)
+    {
+        ::std::cout << "// lang "<< i.first << ::std::endl;
```
Comment (thepowersgang) [inline tools/dump_hirfile/main.cpp:91]: Add the path too

**Comment by catap** [comment]
@thepowersgang here it is

**Pull request merged**